### ISSUE TITLE
pretty high WER

### ISSUE THREAD

**Issue by gezhaoDL** [mravanelli/pytorch-kaldi#199]
HI,sir.I run PyTorch-Kaldi on the THCHS-30 dataset.I used a very simple MLP network trained on mfcc features and got 50.11% WER.Then i tried RNN architectures with mfcc features,the WER was 50.56%.It's pretty high.
I want to know if my profile is wrong.If it is right,how do i lower this? Attached is my configuration file，and by the way, score_thchs30.sh is identical in content to /kaldi/egs/thchs30/s5/local/score.sh.

[THCHS30_RNN_mfcc.log](https://github.com/mravanelli/pytorch-kaldi/files/3999505/THCHS30_RNN_mfcc.log)

**Comment by TParcollet** [comment]
Please, provide us the output of log.log on the exp directory to see how is the loss converging.

**Comment by gezhaoDL** [comment]
> Please, provide us the output of log.log on the exp directory to see how is the loss converging.

Thank you for reply.Here is my log.log
[log.log](https://github.com/mravanelli/pytorch-kaldi/files/4005060/log.log)

**Comment by TParcollet** [comment]
Sorry I meant the res.res file. Also watch out that you have an error at the end of your log.log.

**Comment by mravanelli** [comment]
Yes, it is better to provide the config file and the res.res file

**Comment by gezhaoDL** [comment]
This is the res.res file.
[res.txt](https://github.com/mravanelli/pytorch-kaldi/files/4028624/res.txt)
